src/components/DashStat.container.js

```diff
@@ -27,8 +27,10 @@ export const DashStatTotalTrees = compose(
   connect(
     (state) => ({
       data:
-        state.trees.treeCount !== null
-          ? countToLocaleString(state.trees.treeCount)
+        state.verify.treeCount + state.verify.verifiedTreeCount !== null
+          ? countToLocaleString(
+              state.verify.treeCount + state.verify.verifiedTreeCount,
```

Comment:
`state.verify.treeCount` will change when filters are set in the _Verify_ page, making this total no longer valid (see #28).
Also, this total will not include rejected captures by default, meaning this total will only include unverified and approved captures.

Instead, we need a steady value for each of these stats, like `state.verify.unverifiedTreeCount`.